hack/gen-resources/cmd/commands/cmd.go

```diff
@@ -128,5 +127,6 @@ func NewCleanCommand(opts *util.GenerateOpts) *cobra.Command {
 			}
 		},
 	}
+	command.PersistentFlags().StringVar(&opts.Namespace, "kube-namespace", "argocd", "Name of the namespace where is argocd running [$KUBE_NAMESPACE]")
```

Comment:
nit: just a small change in wording `where argocd is running`

Reply:
Resolved, thank you